src/main/java/com/gregtechceu/gtceu/integration/kjs/GregTechKubeJSPlugin.java

```diff
@@ -271,7 +271,7 @@ public void registerBindings(BindingsEvent event) {
         event.add("RecipeCapability", RecipeCapability.class);
         event.add("ChanceLogic", ChanceLogic.class);
         event.add("CleanroomType", CleanroomType.class);
-        event.add("CraftingComponent", CraftingComponent.class);
+        event.add("CraftingComponent", GTCraftingComponents.class);
```

Comment:
also provide access to the original CraftingComponent class and change this one's name to GTCraftingComponents, this'll probably break KJS anyway.